docs/xlog.md

```diff
@@ -22,21 +22,63 @@
 * Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
 -->
 
-# -Xlog 
+# -Xlog
 
-To control logging to the operating system message log, use [`-Xsyslog`](xsyslog.md).
-- When [`-XX:-LegacyXlogOption`](xxlegacyxlogoption.md) is set, which is the default, the `-Xlog` option is processed for compatibility as described in the explanation.
-- When `-XX:+LegacyXlogOption` is set, `-Xlog` is equivalent to `-Xsyslog`.
+This option controls VM logging and is supported for compatibility with the reference implementation. However, only forms of `-Xlog` that enable garbage collection (GC) logging are recognized.
+
+From Eclipse OpenJ9 version 0.24.0, the [`-Xsyslog`](xsyslog.md) option replaces the existing OpenJ9 `-Xlog`  option for message logging to avoid conflicts with the reference implementation. For backward compatibility, you can control the behavior of the `-Xlog` option with the [`-XX:[+|-]LegacyXlogOption`](xxlegacyxlogoption.md) option.
 
 ## Syntax
 
-        -Xlog:gc[:stderr|:file=<filename>]
+        `-Xlog[:<parameters>]`
+
+Without any parameters, this command runs the `-Xsyslog` option and [`-Xverbosegclog`](xverbosegclog.md). Because the reference implementation of `-Xlog` is intended to enable other forms of logging that are not supported by the OpenJ9 VM, the following warning is printed in addition to enabling verbose GC logging:
+
+```
+JVMJ9VM007W Command-line option unrecognised: <option>
+```
+
+## Parameters
+
+Supported parameters are listed in the sections that follow.
+
+
+### `-Xlog:all`
+
+Setting `-Xlog:all` runs the `-Xsyslog` option and the `-Xverbosegclog` option. Because the reference implementation of `-Xlog` is intended to enable other forms of logging that are not supported by the OpenJ9 VM, the following warning is printed in addition to enabling verbose GC logging:
+
+```
+JVMJ9VM007W Command-line option unrecognised: <option>
+```
+
+### `-Xlog:gc[stderr]`
+
+Setting `-Xlog:gc` on the command line runs the `-Xsyslog` option and `-Xverbosegclog`.
```

Comment:
If `-XX:-LegacyXlogOption` then `-Xlog:<whatever>` has no relation to `-Xsyslog`.
If `-XX:+LegacyXlogOption` then `-Xlog` is the same as `-Xsyslog` where `-Xlog:gc` is not a valid option.